rotkehlchen/chain/avalanche/manager.py

```diff
@@ -58,13 +60,11 @@ def get_avax_balance(self, account: ChecksumEthAddress) -> FVal:
         if result is None:
             balance = from_wei(FVal(self.w3.eth.get_balance(account)))
         else:
-            def filter_avax(value: Dict) -> bool:
-                if 'contract_address' in value:
-                    return value['contract_address'] == AVAX_ADDRESS
-                return False
-
-            avax_coin = list(filter(filter_avax, result))
-            balance = from_wei(FVal(avax_coin[0]['balance']))
```

Comment:
I was afraid of changing this since I don't know if coding it like this was intended, but I believe there is no special restriction

Reply:
If our test works and if you input an avax account and it worked then it should be fine.